sequencing/ATACSeq/README.md

```diff
@@ -0,0 +1,201 @@
+# ATAC-seq analysis pipeline 
+
+This readme explains the content of the scripts for running the ATAC analysis pipeline
+
+REQUISITES:
+- Scripts submitted from the scripts/sequencing folder.
+- The name of the project folder must be specified as the first argument on the command line.
+- Requires config files (file paths should be specified in the first file config.txt):
+  - config.txt with variables for bash scripts
+  - config.r with parameters for R scripts
+  - packagesPip.txt : list of packages and their version to be installed in the created pip virtual environment.
+  - environment.yml : file with packages and their version to be installed by conda.
+- Requires a samples.txt file in the METADATA folder with the names of the samples that will be used to run the pipeline
+- Samples need to be in the RAW folder in the project folder (1_raw)
+- Project directory needs to be specified in the command line (project directory)
+
+Parameters in [] are optional.
+
+
+### 0. Set-up
+
+ `sbatch --array=<number of batch jobs> ATACSeq/jobSubmission/0_setUp.sh (project directory)`
+ 
+This script checks for required files, packages or libraries that are needed later in the pipeline.
+
+##### -scripts executed-
+- ATACSeq/preprocessing/intallLibraries.sh : checks for already installed R libraries and install if not found.
+
+##### -parameters-
+- `--array`: should be 0 for this script as general analysis is run rather than individual samples.
+- `(project-name)` project's directory.
+
+##### -requires-
+- packagesPip.txt, environment.yml and config.r file paths to be specified in the config.txt file.
```

Comment:
Do these need to be specified in the config file, as they are in this repo? If the scripts folder is in the config file, you can build the path from these elements.